src/main/java/baritone/process/MineProcess.java

```diff
@@ -467,8 +469,9 @@ public void mineByName(int quantity, String... blocks) {
     @Override
     public void mine(int quantity, BlockOptionalMetaLookup filter) {
         this.filter = filter;
-        if (filter != null && !Baritone.settings().allowBreak.value) {
-            logDirect("Unable to mine when allowBreak is false!");
+//        baritone.getPathingBehavior().updateIsCloseToHome();
```

Comment:
remove

Reply:
The comment? Think I have removed it in my lateat commit